src/acquisition/covidcast/database.py

```diff
@@ -270,36 +270,60 @@ def compute_covidcast_meta(self, table_name='covidcast', use_index=True):
         srcsigs.put((source, signal))
 
     inner_sql = f'''
-      SELECT
-        `source` AS `data_source`,
-        `signal`,
-        `time_type`,
-        `geo_type`,
-        MIN(`time_value`) AS `min_time`,
-        MAX(`time_value`) AS `max_time`,
-        COUNT(DISTINCT `geo_value`) AS `num_locations`,
-        MIN(`value`) AS `min_value`,
-        MAX(`value`) AS `max_value`,
-        ROUND(AVG(`value`),7) AS `mean_value`,
-        ROUND(STD(`value`),7) AS `stdev_value`,
-        MAX(`value_updated_timestamp`) AS `last_update`,
-        MAX(`issue`) as `max_issue`,
-        MIN(`lag`) as `min_lag`,
-        MAX(`lag`) as `max_lag`
+      SELECT 
+        inner_main_sql.*,inner_min_sql.*     
```

Comment:
do you need all the columns from inner_min_sql to be returned? if not, you can do
```suggestion
        inner_main_sql.*,inner_min_sql.min_issue     
```
-- a column doesn't have to be in the outer SELECT to be used in JOIN .. ON

Reply:
I will fix that problem soon! Thanks for pointing out.